feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,709 @@
+                next-hop: $ATE2_INTERNAL_TE10.v4/32`
+
+*   **GUE Decapsulation:**
+
+    *   For a GUE decapsulation node, configure the following:
+
+        *   UDP port 6080 (configurable) must be used for decapsulating IPv4 and
+            IPv6 payload. The implementation MUST look at the first 4 bits of
+            the UDP payload to determine the GUE version as well as the IP
+            version of the payload to be either IPv4 or IPv6 as explained in the
+            [IETF draft](https://datatracker.ietf.org/doc/html/draft-ietf-intarea-gue-09#section-4)
+        *   The Decapsulation node must be configured for decapsulating traffic
+            received for the range i.e $DUT_TE11.v4/30 and $DUT_TE10.v4/30 in
+            place of the corresponding /32 addresses.
+        *   After decapsulation, the outer TTL and DSCP bits must not be copied
+            to the inner header
+
+### Following Health checks to be run Pre and Post every sub test
+
+Use # Health-1.1: Generic Health Check. If errors identified then the test Must
+fail.
+
+**RT-3.52.1: Baseline**
+
+*   **Situation:**
+
+    *   All IS-IS adjacencies, as well as all IBGP and EBGP peerings, are
+        established.
+    *   No prefixes are exchanged over the IBGP peering between
+        `$ATE2_C.IBGP.v6` and `$DUT_lo0.v6`.
+
+*   **Test Steps:**
+
+    *   Execute all health checks mentioned above prior to running the tests.
+    *   Configure the DUT for IS-IS adjacency, IBGP and EBGP peering, and GUE
+        encapsulation. This includes setting up the necessary routing policies
+        and static routes as described previously.
+    *   Initiate Flow-Set #1 and Flow-Set #2
+    *   Initiate Flow-Set #5
+    *   Send 50000 packets per flow @1000pps
+
+*   **Expectations:**
+
+    *   Flow-Set #1 and Flow-Set #2 are expected to achieve 100% success (zero
+        packet loss) and be routed over the connection between `DUT_Port2` <>
+        `ATE2_Port1`.
+    *   Flow-Set #5 must also achieve 100% success (zero packet loss), with
+        flows routing through the connection between `ATE2_Port1` -->
+        `DUT_Port2` --> `ATE1_Port1`.
+    *   Execute post-test health checks and compare the results with the
+        baseline. Verify that there are no drops, core dumps, or other issues.
+
+**RT-3.52.2: BE1 Traffic Migrated from being routed over the DUT_Port2 -->
+ATE2_Port1 path to DUT_Port4 --> ATE2_Port3 path**
+
+*   **Situation:**
+
+    *   The baseline test (RT-3.52.1) is running, with flows active in
+        Flow-Set #1, Flow-Set #2, and Flow-Set #5
+    *   The DUT receives `$ATE2_INTERNAL_TE11.v4/30` and
+        `$ATE2_INTERNAL_TE10.v4/30` from `$ATE2_Port3`, while advertising
+        `$DUT_TE11.v4/32`, `$DUT_TE10.v4/32`, `$DUT_TE11.v4/30` and
+        `$DUT_TE10.v4/30` to `$ATE2_Port3` via EBGP
+
+*   **Test Steps:**
+
+    *   Execute the health checks described previously
+    *   Configure the DUT according to the prior instructions
```

Comment:
Has the configuration changed?   The DUT config defined in the setup was pushed in 3.52.1 already.  Do you want to change the DUT config in some way and push it again?

Reply:
no, I updated.. it is the same as the baseline config,